src/utilities/omit.js

```diff
@@ -0,0 +1,5 @@
+export default function omit(object = {}, ...keys) {
+  const copy = Object.assign({}, object);
+  keys.forEach(key => delete copy[key]);
+  return copy;
+}
```

Comment:
Cool, changed it now.

> btw, why did you decide to go with in-house implementation instead of lodash's one

lodash.omit was like 30kB unminified, I cant remember exact minified stats. But I felt that for the simple use cases we had we didn't need it.